app/consumer/ante/msg_filter_ante.go

```diff
@@ -54,3 +64,43 @@ func hasValidMsgsPreCCV(msgs []sdk.Msg) bool {
 
 	return true
 }
+
+func fetchMsgExecWrapperMsgs(msgs []sdk.Msg) []sdk.Msg {
+	var wrappedMsgs = []sdk.Msg{}
+	for _, msg := range msgs {
+		msgType := sdk.MsgTypeURL(msg)
+
+		if msgType == "/cosmos.authz.v1beta1.MsgExec" {
+			msgExec, ok := msg.(*authz.MsgExec)
+
+			// TODO: We should decide to continue checking the other msgs or break the loop
+			if !ok {
+				continue
+			}
+
+			sdkMsgs, err := msgExec.GetMessages()
+			if err != nil {
+				continue
+			}
+
+			wrappedMsgs = append(wrappedMsgs, sdkMsgs...)
+
+		}
+	}
+
+	return wrappedMsgs
+}
+
+func hasDisabledModuleMsgs(msgs []sdk.Msg, prefixes ...string) bool {
```

Comment:
Moving this logic to the `DisabledModulesDecorator` would also avoid duplicating this code as it's already there.